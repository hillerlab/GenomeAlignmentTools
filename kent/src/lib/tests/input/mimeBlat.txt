------------0xKhTmLbOuNdArY
Content-Disposition: form-data; name="hgsid"

63932244
------------0xKhTmLbOuNdArY
Content-Disposition: form-data; name="org"

Human
------------0xKhTmLbOuNdArY
Content-Disposition: form-data; name="db"

hg17
------------0xKhTmLbOuNdArY
Content-Disposition: form-data; name="type"

BLAT's guess
------------0xKhTmLbOuNdArY
Content-Disposition: form-data; name="sort"

query,score
------------0xKhTmLbOuNdArY
Content-Disposition: form-data; name="output"

hyperlink
------------0xKhTmLbOuNdArY
Content-Disposition: form-data; name="userSeq"

>NM_000230 (LEP)
gtaggaatcgcagcgccaacggttgcaaggcccaagaagccatcctgggaaggaaaatgc
attggggaaccctgtgcggattcttgtggctttggccctatcttttctatgtccaagctg
tgcccatccaaaaagtccaagatgacaccaaaaccctcatcaagacaattgtcaccagga
tcaatgacatttcacacacgcagtcagtctcctccaaacagaaagtcaccggtttggact
tcattcctgggctccaccccatcctgaccttatccaagatggaccagacactggcagtct
accaacagatcctcaccagtatgccttccagaaacgtgatccaaatatccaacgacctgg
agaacctccgggatcttcttcacgtgctggccttctctaagagctgccacttgccctggg
ccagtggcctggagaccttggacagcctggggggtgtcctggaagcttcaggctactcca
cagaggtggtggccctgagcaggctgcaggggtctctgcaggacatgctgtggcagctgg
acctcagccctgggtgctgaggccttgaaggtcactcttcctgcaaggactacgttaagg
gaaggaactctggcttccaggtatctccaggattgaagagcattgcatggacacccctta
tccaggactctgtcaatttccctgactcctctaagccactcttccaaaggcataagaccc
taagcctccttttgcttgaaaccaaagatatatacacaggatcctattctcaccaggaag
ggggtccacccagcaaagagtgggctgcatctgggattcccaccaaggtcttcagccatc
aacaagagttgtcttgtcccctcttgacccatctccccctcactgaatgcctcaatgtga
ccaggggtgatttcagagagggcagaggggtaggcagagcctttggatgaccagaacaag
gttccctctgagaattccaaggagttccatgaagaccacatccacacacgcaggaactcc
cagcaacacaagctggaagcacatgtttatttattctgcattttattctggatggatttg
aagcaaagcaccagcttctccaggctctttggggtcagccagggccaggggtctccctgg
agtgcagtttccaatcccatagatgggtctggctgagctgaacccattttgagtgactcg
agggttgggttcatctgagcaagagctggcaaaggtggctctccagttagttctctcgta
actggtttcatttctactgtgactgatgttacatcacagtgtttgcaatggtgttgccct
gagtggatctccaaggaccaggttattttaaaaagatttgttttgtcaagtgtcatatgt
aggtgtctgcacccaggggtggggaatgtttgggcagaagggagaaggatctagaatgtg
ttttctgaataacatttgtgtggtgggttctttggaaggagtgagatcattttcttatct
tctgcaattgcttaggatgtttttcatgaaaatagctctttcaggggggttgtgaggcct
ggccaggcaccccctggagagaagtttctggccctggctgaccccaaagagcctggagaa
gctgatgctttgcttcaaatccatccagaataaaacgcaaagggctgaaagccatttgtt
ggggcagtggtaagctctggctttctccgactgctagggagtggtctttcctatcatgga
gtgacggtcccacactggtgactgcgatcttcagagcaggggtccttggtgtgaccctct
gaatgggtccagggttgatcacactctgggtttattacatggcagtgttcctatttgggg
cttgcatgccaaattgtagttcttgtctgattggctcacccaagcaaggccaaaattacc
aaaaatcttggggggtttttactccagtggtgaagaaaactcctttagcaggtggtcctg
agacctgacaagcactgctaggcgagtgccaggactccccaggccaggccaccaggatgc
ccttcccactggaggtcacattcaggaagatgaaagaggaggtttggggtctgccaccat
cctgctgctgtgtttttgctatcacacagtgggtggtggatctgtccaaggaaacttgaa
tcaaagcagttaactttaagactgagcacctgcttcatgctcagccctgactggtgctat
aggctggagaagctcacccaataaacattaagattgaggcctgccctcagggatcttgcg
ttcccagtggtcaaaccgcactcacccatgtgccaaggtggggtatttaccacagcagct
gaacagccaaatgcatggtgcagttgacagcaggtgggaaatggtatgagctgagggggg
ccgtgcccaggggcccacagggaaccctgcttgcactttgtaacatgtttacttttcagg
gcatcttagcttctattatagccacatccctttgaaacaagataactgagaatttaaaaa
taagaaaatacataagaccataacagccaacaggtggcaggaccaggactatagcccagg
tcctctgatacccagagcattacgtgagccaggtaatgagggactggaaccagggagacc
gagcgctttctggaaaagaggagtttcgaggtagagtttgaaggaggtgagggatgtgaa
ttgcctgcagagagaagcctgttttgttggaaggtttggtgtgtggagatgcagaggtaa
aagtgtgagcagtgagttacagcgagaggcagagaaagaagagacaggagggcaagggcc
atgctgaagggaccttgaagggtaaagaagtttgatattaaaggagttaagagtagcaag
ttctagagaagaggctggtgctgtggccagggtgagagctgctctggaaaatgtgaccca
gatcctcacaaccacctaatcaggctgaggtgtcttaagccttttgctcacaaaacctgg
cacaatggctaattcccagagtgtgaaacttcctaagtataaatggttgtctgtttttgt
aacttaaaaaaaaaaaaaaaagtttggccgggtgcggtggctcacgcctgtaatcccagc
actttgggaggccaaggtggggggatcacaaggtcactagatggcgagcatcctggccaa
catggtgaaaccccgtctctactaaaaacacaaaagttagctgagcgtggtggcgggcgc
ctgtagtcccagccactcgggaggctgagacaggagaatcgcttaaacctgggaggcgga
gagtacagtgagccaagatcgcgccactgcactccggcctgatgacagagcgagattccg
tcttaaaaaaaaaaaaaaaaaaagtttgtttttaaaaaaatctaaataaaataactttgc
cccctg


------------0xKhTmLbOuNdArY
Content-Disposition: form-data; name="Submit"

Submit
------------0xKhTmLbOuNdArY
Content-Disposition: form-data; name="seqFile"; filename=""


------------0xKhTmLbOuNdArY--
