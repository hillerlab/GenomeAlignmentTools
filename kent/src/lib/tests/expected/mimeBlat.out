Alternate Header Text:
CONTENT-TYPE:multipart/form-data; boundary=----------0xKhTmLbOuNdArY

content-type: multipart/form-data; boundary=----------0xKhTmLbOuNdArY
nested MIME structure

 content-disposition: form-data; name="hgsid"
 main:[form-data]
 name:[hgsid]
 size:[8]
 data:[63932244]

 content-disposition: form-data; name="org"
 main:[form-data]
 name:[org]
 size:[5]
 data:[Human]

 content-disposition: form-data; name="db"
 main:[form-data]
 name:[db]
 size:[4]
 data:[hg17]

 content-disposition: form-data; name="type"
 main:[form-data]
 name:[type]
 size:[12]
 data:[BLAT's guess]

 content-disposition: form-data; name="sort"
 main:[form-data]
 name:[sort]
 size:[11]
 data:[query,score]

 content-disposition: form-data; name="output"
 main:[form-data]
 name:[output]
 size:[9]
 data:[hyperlink]

 content-disposition: form-data; name="userSeq"
 main:[form-data]
 name:[userSeq]
 size:[3502]
 data:[>NM_000230 (LEP)
gtaggaatcgcagcgccaacggttgcaaggcccaagaagccatcctgggaaggaaaatgc
attggggaaccctgtgcggattcttgtggctttggccctatcttttctatgtccaagctg
tgcccatccaaaaagtccaagatgacaccaaaaccctcatcaagacaattgtcaccagga
tcaatgacatttcacacacgcagtcagtctcctccaaacagaaagtcaccggtttggact
tcattcctgggctccaccccatcctgaccttatccaagatggaccagacactggcagtct
accaacagatcctcaccagtatgccttccagaaacgtgatccaaatatccaacgacctgg
agaacctccgggatcttcttcacgtgctggccttctctaagagctgccacttgccctggg
ccagtggcctggagaccttggacagcctggggggtgtcctggaagcttcaggctactcca
cagaggtggtggccctgagcaggctgcaggggtctctgcaggacatgctgtggcagctgg
acctcagccctgggtgctgaggccttgaaggtcactcttcctgcaaggactacgttaagg
gaaggaactctggcttccaggtatctccaggattgaagagcattgcatggacacccctta
tccaggactctgtcaatttccctgactcctctaagccactcttccaaaggcataagaccc
taagcctccttttgcttgaaaccaaagatatatacacaggatcctattctcaccaggaag
ggggtccacccagcaaagagtgggctgcatctgggattcccaccaaggtcttcagccatc
aacaagagttgtcttgtcccctcttgacccatctccccctcactgaatgcctcaatgtga
ccaggggtgatttcagagagggcagaggggtaggcagagcctttggatgaccagaacaag
gttccctctgagaattccaaggagttccatgaagaccacatccacacacgcaggaactcc
cagcaacacaagctggaagcacatgtttatttattctgcattttattctggatggatttg
aagcaaagcaccagcttctccaggctctttggggtcagccagggccaggggtctccctgg
agtgcagtttccaatcccatagatgggtctggctgagctgaacccattttgagtgactcg
agggttgggttcatctgagcaagagctggcaaaggtggctctccagttagttctctcgta
actggtttcatttctactgtgactgatgttacatcacagtgtttgcaatggtgttgccct
gagtggatctccaaggaccaggttattttaaaaagatttgttttgtcaagtgtcatatgt
aggtgtctgcacccaggggtggggaatgtttgggcagaagggagaaggatctagaatgtg
ttttctgaataacatttgtgtggtgggttctttggaaggagtgagatcattttcttatct
tctgcaattgcttaggatgtttttcatgaaaatagctctttcaggggggttgtgaggcct
ggccaggcaccccctggagagaagtttctggccctggctgaccccaaagagcctggagaa
gctgatgctttgcttcaaatccatccagaataaaacgcaaagggctgaaagccatttgtt
ggggcagtggtaagctctggctttctccgactgctagggagtggtctttcctatcatgga
gtgacggtcccacactggtgactgcgatcttcagagcaggggtccttggtgtgaccctct
gaatgggtccagggttgatcacactctgggtttattacatggcagtgttcctatttgggg
cttgcatgccaaattgtagttcttgtctgattggctcacccaagcaaggccaaaattacc
aaaaatcttggggggtttttactccagtggtgaagaaaactcctttagcaggtggtcctg
agacctgacaagcactgctaggcgagtgccaggactccccaggccaggccaccaggatgc
ccttcccactggaggtcacattcaggaagatgaaagaggaggtttggggtctgccaccat
cctgctgctgtgtttttgctatcacacagtgggtggtggatctgtccaaggaaacttgaa
tcaaagcagttaactttaagactgagcacctgcttcatgctcagccctgactggtgctat
aggctggagaagctcacccaataaacattaagattgaggcctgccctcagggatcttgcg
ttcccagtggtcaaaccgcactcacccatgtgccaaggtggggtatttaccacagcagct
gaacagccaaatgcatggtgcagttgacagcaggtgggaaatggtatgagctgagggggg
ccgtgcccaggggcccacagggaaccctgcttgcactttgtaacatgtttacttttcagg
gcatcttagcttctattatagccacatccctttgaaacaagataactgagaatttaaaaa
taagaaaatacataagaccataacagccaacaggtggcaggaccaggactatagcccagg
tcctctgatacccagagcattacgtgagccaggtaatgagggactggaaccagggagacc
gagcgctttctggaaaagaggagtttcgaggtagagtttgaaggaggtgagggatgtgaa
ttgcctgcagagagaagcctgttttgttggaaggtttggtgtgtggagatgcagaggtaa
aagtgtgagcagtgagttacagcgagaggcagagaaagaagagacaggagggcaagggcc
atgctgaagggaccttgaagggtaaagaagtttgatattaaaggagttaagagtagcaag
ttctagagaagaggctggtgctgtggccagggtgagagctgctctggaaaatgtgaccca
gatcctcacaaccacctaatcaggctgaggtgtcttaagccttttgctcacaaaacctgg
cacaatggctaattcccagagtgtgaaacttcctaagtataaatggttgtctgtttttgt
aacttaaaaaaaaaaaaaaaagtttggccgggtgcggtggctcacgcctgtaatcccagc
actttgggaggccaaggtggggggatcacaaggtcactagatggcgagcatcctggccaa
catggtgaaaccccgtctctactaaaaacacaaaagttagctgagcgtggtggcgggcgc
ctgtagtcccagccactcgggaggctgagacaggagaatcgcttaaacctgggaggcgga
gagtacagtgagccaagatcgcgccactgcactccggcctgatgacagagcgagattccg
tcttaaaaaaaaaaaaaaaaaaagtttgtttttaaaaaaatctaaataaaataactttgc
cccctg

]

 content-disposition: form-data; name="Submit"
 main:[form-data]
 name:[Submit]
 size:[6]
 data:[Submit]

 content-disposition: form-data; name="seqFile"; filename=""
 main:[form-data]
 name:[seqFile]
 filename:[]
 size:[0]
 data:[]

